com.unity.perception/Runtime/GroundTruth/Labeling/IdLabelEntry.cs

```diff
@@ -16,5 +16,26 @@ public struct IdLabelEntry : ILabelEntry
         /// The id to associate with the label.
         /// </summary>
         public int id;
+
+        /// <inheritdoc/>
+        public bool Equals(IdLabelEntry other)
+        {
+            return label == other.label && id == other.id;
```

Comment:
Doesn't need to be addressed in this PR, but can we end up with a label that has the same string as another label but a different ID, or same ID but different strings?